web/themes/joinup/prototype/sass/components/_ec-menu.sass

```diff
@@ -0,0 +1,33 @@
+// European Commision menu
```

Comment:
Added footer under number 2.14 in the style guide. I divided it into two components: EC menu and social media menu. I think that showing it as one component in this default, narrow style guide view might be confusing.